rets/client/decoder.py

```diff
@@ -67,9 +68,13 @@ def _get_decoder(data_type: str, interpretation: str, include_tz: bool = False):
 
 
 def _decode_datetime(value: str, include_tz: bool) -> datetime:
-    # Correct `0000-00-00 00:00` to `0000-00-00T00:00`
-    if value[10] == ' ':
+    # Correct `0000-00-00 00:00:00` to `0000-00-00T00:00:00`
+    if re.match(r'^\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2}$', value):
```

Comment:
I think dateutil.parser.parse is more robust than regex matching:

from datetutil.parser import parse

parse('1970-01-01') will return datetime.datetime(1970, 1, 1, 0, 0)
parse('1970-01-01 10:11) will return datetime.datetime(1970, 1, 1, 10, 11)